WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -1,77 +1,79 @@
 <% provide(:title, "Registrations for #{@competition.name}") %>
 
 <%= render layout: "nav" do %>
-  <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
-    <thead>
-      <tr>
-        <th class="name" data-sortable="true" data-field="name">Name</th>
-        <th class="wca-id" data-sortable="true">WCA ID</th>
-        <th class="country" data-sortable="true">Citizen of</th>
+  <% cache ["registrations_index", @registrations.pluck(:id), @registrations.maximum(:updated_at)] do %>
```

Comment:
1. Do we care about the order of `@registrations`? If not, I'd use `Set.new(@registrations.pluck(:id))`.

2. We're also using `@competition.events` inside the cached block. Assuming they can change, they should be in the cache key.